### DISCUSSION TITLE
Is this a real problem?

### DISCUSSION THREAD

**Discussion by hammyau** [eclipse-tycho/tycho#428] [Q&A] [answered]
As part of #418 I have cloned the repo and have built it.

There were a couple of issues along the way.
I have the latest Eclipse for Committers.
I had already cloned this repo.
I then imported the Maven projects.
Eclipse gets itself into a mess endlessly looping of what seem to be circular resolving issues. I think this is because the automatic build was on. I stopped the jobs and switched of the autobuild.

I then followed the instructions and set the tycho project as the active target. Ran  maven install and all good I have a 2.6.0-SNAPSHOT in my .m2 repository.

However, Eclipse is still left with 192  problems.
A mixture of issues.

`Execution default of goal biz.aQute.bnd:bnd-maven-plugin:6.1.0:bnd-process failed: Plugin biz.aQute.bnd:bnd-maven-plugin:6.1.0 or one of its dependencies could not be resolved: The following artifacts could not be resolved: biz.aQute.bnd:biz.aQute.resolve:jar:6.1.0, biz.aQute.bnd:biz.aQute.repository:jar:6.1.0, org.eclipse.sisu:org.eclipse.sisu.plexus:jar:0.3.2, org.eclipse.sisu:org.eclipse.sisu.inject:jar:0.3.2, org.apache.maven:maven-repository-metadata:jar:3.3.9, org.apache.maven:maven-settings:jar:3.3.9, org.eclipse.aether:aether-api:jar:1.0.2.v20150114, biz.aQute.bnd:biz.aQute.bndlib:jar:6.1.0, org.osgi:org.osgi.util.tracker:jar:1.5.4, org.osgi:org.osgi.service.log:jar:1.3.0: Could not transfer artifact biz.aQute.bnd:biz.aQute.resolve:jar:6.1.0 from/to central (https://repo.maven.apache.org/maven2): The operation was cancelled. (biz.aQute.bnd:bnd-maven-plugin:6.1.0:bnd-process:default:process-classes)`

And a bunch of 

`org.apache.maven.plugin.PluginExecutionException: Execution default-resources of goal org.apache.maven.plugins:maven-resources-plugin:3.2.0:resources failed: Plugin org.apache.maven.plugins:maven-resources-plugin:3.2.0 or one of its dependencies could not be resolved: Failed to collect dependencies at org.apache.maven.plugins:maven-resources-plugin:jar:3.2.0 -> org.apache.maven:maven-core:jar:3.1.0 -> org.apache.maven:maven-settings-builder:jar:3.1.0 -> org.codehaus.plexus:plexus-interpolation:jar:1.16`

**Comment by laeubi** [comment, marked as answer]
If the maven build succeed everything is fine.

About the IDE setup, the tycho workspace is sadly not that straigth forward and we are still in the progress of improving it. First of all you can ignore any of the "Execution not covered" e.g by choosing "ignore in workspace settings".

"the operation was cancelled" might indicate a network or proxy problem, but here again it seems not harmful as it only affects the incremental build in the IDE, as tycho is hardly runnable from the IDE itself this dosen't matter much here.

Anyways I wonder why you need to build a custom tycho version for porting your application? Is there any issue you want to debug/fix?

**Comment by hammyau** [comment]
I am just building it in an effort to understand what it is doing and how it works.

**Reply by laeubi** [reply]
If there are any specific questions feel free to open a Q&A in the discussions as well.

**Comment by hammyau** [comment]
See my comment in #418

**Comment by hammyau** [comment]
I'm unsure whether to add replies or new comments?
